src/instance.ts

```diff
@@ -164,6 +165,7 @@ class Instance extends common.GrpcServiceObject {
   databases_: Map<string, Database>;
   metadata?: IInstance;
   resourceHeader_: {[k: string]: string};
+  observabilityConfig: ObservabilityOptions | undefined;
```

Comment:
we should define observabilityConfig as optional using "?" instead of using "undefined". Same standard should be used everywhere. 

Also use param name `observabilityOptions` same as the type name.

Reply:
Similar to Database , use underscore to mark this property as private.

Reply:
Done, I've marked it as private.

Reply:
Done.